service/src/tests/mod.rs

```diff
@@ -33,6 +31,7 @@ pub mod parentchain_block_syncer_test;
 pub fn run_enclave_tests(matches: &ArgMatches) {
 	println!("*** Starting Test enclave");
 	let config = Config::from(matches);
+	setup::purge_files_from_cwd().unwrap();
```

Comment:
Where's that coming from?

Reply:
We did a cleanup in an earlier PR, that removed dependencies on the state handler, which implicitly did a state cleanup when we ran these tests. On CI that works fine, because we always have a clean directory. But locally on my machine, I ran into test failures, because previous states were not cleaned up properly. So I had to introduce this purge.